project/Github.scala

```diff
@@ -11,6 +11,6 @@ object GitHub {
 
   def url(v: String): String = {
     val branch = if (v.endsWith("SNAPSHOT")) "master" else "v" + v
-    "http://github.com/akka/akka/tree/" + branch
+    "http://github.com/akka/akka-http/tree/" + branch
```

Comment:
Yes, although this is not yet used in the docs. Ideally all links outside the Paradox universe to files in the repo should automatically link to some repo, maybe taken from the SBT `ScmInfo`.